py-polars/tests/parametric/test_groupby_rolling.py

```diff
@@ -41,7 +41,7 @@ def test_group_by_rolling(
             ],
         )
     )
-    df = dataframe.sort("ts").unique("ts")
+    df = dataframe.sort("ts")
```

Comment:
this `unique` can be removed now 🥳 (btw, my fault here for not having originally left a comment about why it was there)